### PR TITLE
feat: Adding hover behaviour on select components

### PR DESCRIPTION
**Pull request by itssharmasandeep**
## Description
This PR adds hover and pressed behaviour on select, multi-select and menu-dropdown buttons

### Testing
Tested locally

Screenshot for multi-select pressed
Screenshot 2021-03-31 at 3 44 04 PM

### Checklist:
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by arjunlalb** [comment]
@itssharmasandeep Could you also add screenshots/screen recording of the behaviour?

File: projects/components/src/select/select.component.ts [hunk 1]
```diff
@@ -61,7 +64,7 @@ import { SelectSize } from './select-size';
             <div
               *ngSwitchCase="'${SelectTriggerDisplayMode.Icon}'"
               class="trigger-content icon-only"
-              [ngClass]="this.selected !== undefined ? 'selected' : ''"
+              [ngClass]="[this.selected !== undefined ? 'selected' : '', this.popoverOpen ? 'open' : '']"
```
Comment (arjunlalb) [inline projects/components/src/select/select.component.ts:67]: may be we should make this an object for better readability

`"{ selected: this.selected !== undefined, open: this.popoverOpen}"`

**Review by arjunlalb — DISMISSED** [review]

File: projects/components/src/select/select.component.scss [hunk 2]
```diff
@@ -99,6 +104,10 @@
     display: flex;
     align-items: center;
 
+    &.open {
+      background-color: transparent;
```
Comment (aaron-steinfeld) [inline projects/components/src/select/select.component.scss:108]: Is this doing anything or is it overridden by line 124 (indentation may be throwing me off here)

File: projects/components/src/select/select.component.ts (hunk 1 above)
Comment (itssharmasandeep) [reply projects/components/src/select/select.component.ts]: done

File: projects/components/src/select/select.component.scss (hunk 2 above)
Comment (itssharmasandeep) [reply projects/components/src/select/select.component.scss]: done

**Review by aaron-steinfeld — APPROVED** [review]

**Review by arjunlalb — APPROVED** [review]

**Pull request merged**